.github/libs/CONST.ts

```diff
@@ -9,9 +9,11 @@ const CONST = {
         INTERNAL_QA: 'InternalQA',
     },
     DATE_FORMAT_STRING: 'yyyy-MM-dd',
+    APP_REPO_URL: '',
+    APP_REPO_GIT_URL: '',
 };
 
 CONST.APP_REPO_URL = `https://github.com/${CONST.GITHUB_OWNER}/${CONST.APP_REPO}`;
 CONST.APP_REPO_GIT_URL = `git@github.com:${CONST.GITHUB_OWNER}/${CONST.APP_REPO}.git`;
```

Comment:
we cant do think because we are using variables from object where we are defining those fields
<img width="1144" alt="image" src="https://github.com/Expensify/App/assets/25230417/b61c14be-d174-42c8-a92e-908803c781d3">

Reply:
oh duhhhhh 👍 😅